src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx

```diff
@@ -0,0 +1,546 @@
+---
+title: Monitor your .NET MAUI mobile application
+tags:
+    - .NET MAUI
+    - Mobile monitoring
+    - Hybrid framework
+    - Get started
+metaDescription: "New Relic's .NET MAUI agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+---
+
+import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.webp'
+
+Our New Relic .NET MAUI agent monitors your .NET MAUI mobile app and provides deep insights into your app's performance, errors, and user experience. With the agent, you can:
+
+* Capture C# errors to quickly identify and fix problems.
+* Track network requests to see how your app is interacting with the backend.
+* Use distributed tracing to drill down into handled exceptions and identify the root cause of problems.
+* Create custom tracking events and metrics to fully understand how your users are interacting with your app.
+
+<img
+  title="Mobile summary view in the UI"
+  alt="Summary view of a .NET MAUI mobile app in New Relic"
+  src={mobileFlutterSummary}
+  style={{ maxWidth: '70%' }}
+/>
+
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select your .NET MAUI app) > Summary**: View .NET MAUI data, track HTTP requests and errors, and monitor how your app is performing over time.
+</figcaption>
+
+## Install the .NET MAUI SDK
+<Steps>
+
+<Step>
+### Review the requirements
+
+Before you install our .NET MAUI agent, make sure your app meets these version requirements:
+* .NET version 7.0 and higher
+* For Android-native apps: Android 7 (API 24) or higher
+* For iOS-native apps:
+    * iOS 11 or higher
+    * Latest release of Xcode
+</Step>
+
+<Step>
+### Add the .NET MAUI agent to your project
+
+First, you'll need to add our agent, a NuGet package, to your MAUI project:
+
+1. Open your .NET MAUI solution, select the project you want to add the agent to, and open its context menu.
+2. Click **Add > Add NuGet packages**, then select `NewRelic.MAUI.Plugin`.
+
+</Step>
+
+<Step>
+### Copy your application token from the UI [#app-token]
+
+The application token is used for New Relic to authenticate your .NET MAUI app's data. 
+
+To view and copy your application token in the New Relic UI:
```

Comment:
This might be a case where we don't have to document what's already described elsewhere. We could say something like: 

Get your application token from the New Relic mobile UI, on the **Installation** or **Application** page. For more detail, see Application token...